res/controllers/Numark_DJ2GO2_Touch_scripts.js

```diff
@@ -91,7 +91,7 @@ DJ2GO2Touch.Deck = function(deckNumbers, midiChannel) {
         key: "super1"
     });
 
-    engine.setValue(this.currentDeck, "rate_dir", -1);
+    engine.setValue(this.currentDeck, "rate_dir", 1);
```

Comment:
I don't think this is correct. This is actually user-configurable and should not be touched by the hardware mapping. What if you instead specify `invert: true` on the `tempoFader`? Thats what I had to do on my Numark Controller.